src/test/java/Pages/AddToCart.java

```diff
@@ -0,0 +1,29 @@
+//package Pages;
+//
+//import org.openqa.selenium.By;
+//import org.openqa.selenium.Keys;
+//import org.openqa.selenium.WebDriver;
+//
+//public class AddToCart {
+//
+//    protected WebDriver driver;
+//
```

Comment:
oare clasa asta de ce e comentata?

Reply:
Pentru ca a ramas neimplementata. Imi dadea erori cand le-am despartit. Urmeaza sa iau ce nu tine de login si sa pun in AddToCart.